deploy-as-code/helm/charts/health-services/plan-service/values.yaml

```diff
@@ -93,3 +93,5 @@ env: |
   {{- end }}
   - name: RESOURCE_CONFIG_CONSUMER_PLAN_CREATE_TOPIC
     value: "resource-microplan-create-topic"
+  - name: ROLE.MAP
```

Comment:
why is dot present here? it should be underscore

Reply:
yaa...correcting it